_layouts/intrapreneur-role.html

```diff
@@ -26,9 +26,9 @@ <h3 class="fr-card__title">
                         </h3>
                         <div class="fr-card__description">
                         <ol>
-                            <li><a href="#l-intrapreneur-vient-du-terrain">L’intrapreneur.e vient du terrain</a></li>
-                            <li><a href="#au-coeur-d-une-equipe-produit">Au coeur d’une équipe produit</a></li>
-                            <li><a href="#accompagne-du-pres">Accompagné du près</a></li>
+                            <li><a href="#l-intra-vient-du-terrain">L’intra vient du terrain</a></li>
```

Comment:
Je crois que tu as cassé le lien
```suggestion
                            <li><a href="#l-intrapreneur-vient-du-terrain">L’intra vient du terrain</a></li>
```